cmdutil/closureLogic_test.go

```diff
@@ -0,0 +1,59 @@
+package cmdutil
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/dachad/tcpgoon/mtcpclient"
+	"github.com/dachad/tcpgoon/tcpclient"
+)
+
+func TestCloselyNicelyPendingConnections(t *testing.T) {
```

Comment:
You can add more works in the middle to make this more readable. For instance: TestCloselyNicelyWhenThereArePendingConnections

Reply:
In fact, TestCloselyNicely does not make too much sense xD